javascript/javascript1/week1/script.js

```diff
@@ -0,0 +1,103 @@
+// FreeCodeCamp user: https://www.freecodecamp.org/catalina-tech
+
+
+
+// Age-ify (A future age calculator)
+
+let yearOfBirth;
+yearOfBirth = 1988;
+console.log(typeof yearOfBirth);
+
+let yearFuture;
+yearFuture = 2028;
+console.log(typeof yearFuture);
+
+let age = yearFuture - yearOfBirth;
+console.log("You will be" + " " + age + " " + "in" + " " + yearFuture); 
+
+
+
+
+//Goodboy-Oldboy (A dog age calculator)
+
+let dogYearOfBirth;
+dogYearOfBirth = 2017;
+
+let dogYearFuture;
+dogYearFuture = 2027;
+
+//Human years
+let dogYear = dogYearFuture - dogYearOfBirth;
+
+//Dog Years
+let dogYearOld = dogYear*7;
+
+let shouldShowResultInDogYears = false;
+
+if (shouldShowResultInDogYears) {
+    console.log("Your dog will be" + " " + dogYearOld + " " + "dog years old in " + dogYearFuture);    
+} else {
+    console.log("Your dog will be" + " " + dogYear + " " + "human years in " + dogYearFuture);  
+};
+
+
+
+
+//Housey pricey (A house price estimator)
+
+let volumeInMeters;
+let gardenSizeinM2;
+let housePrice;
+housePrice = (volumeInMeters * 2.5 * 1000) + (gardenSizeinM2 * 300);
+
+let peterHouseWide = 8;
+let peterHouseDeep = 10;
+let peterHouseHigh = 10;
+let peterGardenSize = 100;
+let peterHousePrice = 2500000;
+let volumeInMetersPeterHouse = (peterHouseDeep*peterHouseHigh*peterHouseWide);
+
+let peterHouse;
+peterHouse = (volumeInMetersPeterHouse * 2.5 * 1000) + (peterGardenSize * 300);
+peterHouseExtraMoney = 2500000 - peterHouse;
+//console.log(peterHouse);
+
+if(peterHouse < 2500000) {
```

Comment:
Same here, you can use your predefined variable here:
```suggestion
if(peterHouse < peterHousePrice) {
```